lib/decidim/term_customizer/context/job_context.rb

```diff
@@ -10,6 +10,8 @@ def resolve!
           # passed for the job.
           user = nil
           data[:job].arguments.each do |arg|
+            arg = arg[:args].first if arg.is_a?(Hash) && arg.has_key?(:args)
```

Comment:
I would suggest adding a comment(i.e with a reference to https://github.com/mainio/decidim-module-term_customizer/issues/93)

Reply:
Thank you for your review, I convert to draft and re-open it once I implemented your suggestion !

Reply:
I can move this line in `organization_from_argument` method but I wonder if we should also keep it for `space_from_argument` and `component_from_argument` because all these methods takes `arg` as argument